apps/ui/package.json

```diff
@@ -27,12 +27,6 @@
     "@casl/ability": "^6.5.0",
     "@casl/prisma": "^1.4.1",
     "@casl/vue": "^2.1.2",
-    "@fortawesome/fontawesome-free": "^6.4.0",
-    "@fortawesome/fontawesome-svg-core": "^6.1.1",
-    "@fortawesome/free-brands-svg-icons": "^6.1.1",
-    "@fortawesome/pro-duotone-svg-icons": "^6.1.1",
-    "@fortawesome/pro-light-svg-icons": "^6.1.1",
-    "@fortawesome/pro-solid-svg-icons": "^6.1.2",
```

Comment:
These needed to be removed when you ran `npm i` but we actually still need them in our `package.json`. again, working on a fix for this! 🙈